test/complexrestriction.cc

```diff
@@ -1,5 +1,7 @@
 
 #include "baldr/complexrestriction.h"
+#include "baldr/graphid.h"
+#include "midgard/logging.h"
```

Comment:
logging header seems accidental?

Reply:
I actually did log stuff while debugging but have since removed the debug statements. Would be nice with the tool @danpat used for automatically cleaning up un-needed includes.